ios/FluentUI/Core/Theme/FluentTheme+visionOS.swift

```diff
@@ -14,7 +14,7 @@ extension FluentTheme {
         // need to provide multiple values (e.g. light + dark, elevated, etc).
         switch token {
         case .foreground1:
-            visionColorDark = GlobalTokens.neutralColor(.white)
+            visionColorDark = GlobalTokens.neutralSwiftUIColor(.white)
```

Comment:
wait I thought Mike fixed this, can we just make it .white

Reply:
I think you still need to specify if it's neutralColor token or shared, etc...

Reply:
no like just skip GlobalTokens altogether, just visionColorDark = .white. like how foreground2 does it

Reply:
Well .white is the system Color.white, not .white from our global tokens. Do we know why FG1 uses the token white and the others use system white? :D

Reply:
We talked offline and I finally understood that I am an idiot.